paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -313,10 +313,10 @@ public void createTable(Identifier identifier, Schema schema, boolean ignoreIfEx
                     e);
         }
         Table table = newHmsTable(identifier);
-        updateHmsTable(table, identifier, tableSchema);
         try {
+            updateHmsTable(table, identifier, tableSchema);
```

Comment:
The method `updateHmsTable` will not generate schema file in the table location?

Reply:
And can you add test for this PR?

Reply:
the method `updateHmsTable `. do not generate schema file ,  but it may throw an exception , we do not try catch it .

and I add a test case